config/initializers/apipie.rb

```diff
@@ -39,7 +39,7 @@
   config.ignored = []
   config.ignored_by_recorder = %w[]
   config.doc_base_url = "/apidoc"
-  config.use_cache = Rails.env.production? || File.directory?(config.cache_dir)
+  config.use_cache = false
```

Comment:
For some it might be useful to change this setting in their dev env. Maybe a comment will be handy:

```ruby
# By default we disabled the cache, but if you need to speed the testing up, you can switch it to `true`.
# To speed up the generation of the cache, please use FOREMAN_APIPIE_LANGS=en bundle exec rake apipie:cache
```